variables.tf

```diff
@@ -72,6 +72,12 @@ variable "image" {
   description = "Docker image to run in the ECS cluster"
 }
 
+variable "kms_key_id" {
+  type        = string
+  default     = null
+  description = "The custom KMS key ID used encryption of the Cloudwatch log group"
```

Comment:
Is this indeed used for CloudWatch? Or is this a copy/past thingy?

Reply:
Nope, that's for Cloudwatch (https://aquasecurity.github.io/tfsec/latest/checks/aws/cloudwatch/log-group-customer-key/)

`CloudWatch log groups are encrypted by default, however, to get the full benefit of controlling key rotation and other KMS aspects a KMS CMK should be used.`